Employee_details_using _structures/EmployeeDetails.c

```diff
@@ -0,0 +1,406 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<stdbool.h>
+#include<string.h>
+#include"EmployeeDetails.h"	//why EmployeeDetails"1"? // name changed
+#define NAME_LENGTH 10   //Naming to be more specific ///naming changed
+enum menu{ADD=1,VIEW,SORT_BY_NAME,REMOVE_BY_ID,SORT_BY_ID,REMOVE_BY_NAME,EXIT};
+bool string_validation(char *employee_name)
+{
+	int string_length=strlen(employee_name),indexOfchar=0;
+	if(string_length>NAME_LENGTH)
+	{
+		printf("more than 10 characters!!\n");
+		return false;
+	}
+	while(indexOfchar<string_length)
+		{
+			if((employee_name[indexOfchar]>='A' && employee_name[indexOfchar]<='Z')||( employee_name[indexOfchar]>='a' && employee_name[indexOfchar]<='z'))

```

Comment:
What should be the maximum number of characters in a single line

Reply:
Maximum 78 characters per line.I have changed it in my recent commit.